vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -90,6 +90,7 @@ def load_hosts
       def parse_host(host, indexed_servers, _indexed_hosts_ports, indexed_resources)
         uid = host.uuid
         host_name = identify_host_name(indexed_resources, host.instance_uuid, uid)
+        ip_address = identify_primary_ip_address(host, indexed_servers)
```

Comment:
We have been switching to hostname ever, but this seems to imply switch to ip address. @jrafanie ?